packages/website/pages/terms.js

```diff
@@ -23,75 +23,161 @@ export default function TermsOfService() {
         <p className="lh-copy">
           The following terms and conditions govern all use of the{' '}
           <a className="black" href="https://nft.storage">
-            nft.storage
+            NFT.Storage
           </a>{' '}
-          website (the “Website”) and all content, services and products
-          available at or through the Website. The Website is offered subject to
-          your acceptance without modification of all of the terms and
-          conditions contained herein. As all data uploaded to nft.storage will
+          website (the “Website”) and all content, services, and products
+          available at or through the Website. As all data uploaded to {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} will
           be stored on IPFS, this website incorporates the{' '}
           <a className="black" href="https://discuss.ipfs.io/tos">
             Terms of Service of IPFS.io
           </a>
           .
         </p>
-        <p className="lh-copy">
-          If you do not agree to all the terms and conditions of this agreement,
-          then you may not access the Website or use any services.
-        </p>
+        
         <h2 className="chicagoflf">
           <HashLink id="storage-term">Storage Term</HashLink>
         </h2>
         <p className="lh-copy">
-          Data will be stored at no cost to the user on IPFS for as long as
-          Protocol Labs, Inc. continues to offer free storage for NFT’s. Data
-          will continue to be persisted ad infinitum <strong>or</strong> until
-          Protocol Labs decides to conclude the NFT.storage project. Prior to
-          termination, Protocol Labs will provide 90 days notice to users via
-          email to allow users enough time to make arrangements for storing
-          their data by other means.
+          All data is stored redundantly on both the IPFS and Filecoin networks, 
+          and guaranteed to be publicly available and verifiable via its unique 
+          content identifier.
+        </p>
+        <p className="lh-copy">
+          All data stored on {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} is stored in long-term (18 month) deals 
+          with independent Filecoin Storage Providers. Filecoin storage deals can 
+          be renewed by anyone and are non-custodial. Anyone can access information 
```

Comment:
> Filecoin storage deals can be renewed by anyone

Is this right? I know you can create a deal with another storage provider for the same data, but I wasn't aware that renewal in the sense we're alluding to here was possible yet?

Reply:
Making a new deal is the only way to renew whether you were the original dealmaker or not. So it's a matter of whether we want to use the term "renew" at all - IMHO the mechanism qualifies as renewal but open to discussion!

Reply:
> Making a new deal is the only way to renew ...

Note: the above is true **today**. Unclear where we will be 1 year from now ( most deals would not have expired yet ).